server/chat-plugins/room-events.ts

```diff
@@ -8,9 +8,24 @@
  */
 import {Utils} from '../../lib/utils';
 
-type RoomEvent = NonNullable<import('../rooms').RoomSettings['events']>[''];
+export interface RoomEvent {
+	eventName: string;
+	date: string;
+	desc: string;
+	started: boolean;
+}
+
+export interface RoomEventAlias {
+	id: ID;
```

Comment:
Hm, I think it's confusing for aliases/categories to have `id` properties, but not events. I'd probably omit `id` from all of them - that prevents `id` from getting out of sync with the `room.settings.events` key, after all.

Reply:
Alright.